lib/protip/standard_converter.rb

```diff
@@ -23,6 +44,13 @@ class << self
       end
     }
 
+    @conversions['protip.messages.Range'] = {
```

Comment:
this looks like it doesn't belong in this PR, can you confirm if it's needed here?